packages/playground/src/explorer/views/farms.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <view-layout>
+    <v-row>
+      <v-col>
+        <filters
+          v-model="filterFarmInputs"
+          @update:model-value="inputFiltersReset"
+          :form-disabled="isFormLoading"
+          v-model:valid="isValidForm"
+        />
+        <FarmsTable :items="farms" :loading="loading" :selectedFarm="selectedFarm" @open-dialog="openDialog" />
+      </v-col>
+    </v-row>
+    <farmDialog v-if="selectedFarm" :openDialog="isDialogOpened" :farm="selectedFarm" @close-dialog="closeDialog" />
+  </view-layout>
+</template>
+
+<script lang="ts" setup>
+import type { Farm } from "@threefold/gridproxy_client";
+import debounce from "lodash/debounce.js";
+import { onMounted, ref, watch } from "vue";
+
+import type { FilterFarmInputs } from "../../utils/filter_farms";
+import { inputsInitializer } from "../../utils/filter_farms";
+import { getFarmQueries, getFarms } from "../utils/helpers";
+import type { FarmFilterOptions, MixedFarmFilter } from "../utils/types";
+import { farmOptionsInitializer } from "../utils/types";
+
+const loading = ref<boolean>(false);
+const farms = ref<Farm[]>();
+const isDialogOpened = ref<boolean>(false);
+const selectedFarm = ref<Farm>();
+const filterFarmInputs = ref<FilterFarmInputs>(inputsInitializer);
+const filterOptions = ref<FarmFilterOptions>(farmOptionsInitializer);
+const mixedFarmFilters = ref<MixedFarmFilter>({ inputs: filterFarmInputs.value, options: filterOptions.value });
+const isFormLoading = ref<boolean>(true);
+const isValidForm = ref<boolean>(false);
+
+const _getFarms = async (queries: Partial<FarmsQuery>) => {
+  loading.value = true;
+  isFormLoading.value = true;
+
+  const { count, data } = await getFarms(queries);
```

Comment:
Can we do the query in a try & catch? In case catch entered, we can show a toast to the user with the error. check createCustomToast.